app/views/inventory/manual.snapshot.controller.js

```diff
@@ -108,13 +108,24 @@ function snapshotManualCtrl ($scope, android, fields, $state, enums, resources)
         new fields.Select('device.chassis', {
           namespace: ns,
           options: enums.Chassis.options(fields),
-          required: true,
```

Comment:
Currently for the Computers subtypes the chassis attribute is not allowed to be null, but it should not be. The way to solve this would be to let the chassis attribute be null. 
More specifically, here is the related [issue](https://github.com/eReuse/devicehub-teal/issues/93) on DHTeal. 

**In any case it is good that the chassis is not a required field from the Client.**